core/sql/executor/HiveClient_JNI.cpp

```diff
@@ -42,17 +43,16 @@ static const char* const hvcErrorEnumStr[] =
  ,"Java exception in close()."
  ,"Preparing parameters for exists()."
  ,"Java exception in exists()."
- ,"Preparing parameters for getHiveTableStr()."
- ,"Java exception in getHiveTableStr()."
- ,"Preparing parameters for getHiveTableParameters()."
- ,"Java exception in getHiveTableParameters()."
  ,"Preparing parameters for getRedefTime()."
  ,"Java exception in getRedefTime()."
  ,"Java exception in getAllSchemas()."
  ,"Preparing parameters for getAllTables()."
  ,"Java exception in getAllTables()."
  ,"Preparing parameters for executeHiveSQL()."
  ,"Java exception in executeHiveSQL()."
+ ,"Preparing parameters for getHiveTableInfo()."
+ ,"Java exception in getHiveTableInfo()."
+ ,"Error in getHiveTableInfoDetails()."
```

Comment:
Should there also be error text for HVC_ERROR_POPULATE_SDS_ERROR? More generally, the entries in this array of strings do not match in number those of the enum HVC_RetCode in HiveClient_JNI.h.

Reply:
Yes. Corrected and  ensured that it matches